GenderPayGap.WebUI/Views/Viewing/Download/Download.cshtml

```diff
@@ -30,15 +32,18 @@
         <hr/>
 
         <ul class="govuk-list">
-            @foreach (int year in ReportingYearsHelper.GetReportingYears().OrderByDescending(y => y))
-            {
-                <li>
-                    <a href="@Url.Action("DownloadData", "Viewing", new {year})"
-                       class="govuk-link">
-                        <span class="govuk-visually-hidden">Download gender pay gap data for</span>
-                        Reporting year @(ReportingYearsHelper.FormatYearAsReportingPeriod(year)) (CSV file)
-                    </a>
-                </li>
+            @{
+                List <(int,string)> reportingYearsWithFileSizes = ReportingYearsHelper.GetReportingYearWithCSVFileSize();
+                foreach (var item in reportingYearsWithFileSizes)
+                {
+                    <li>
+                        <a href="@Url.Action("DownloadData", "Viewing", item.Item1)"
+                           class="govuk-link">
+                            <span class="govuk-visually-hidden">Download gender pay gap data for</span>
+                            Reporting year @(ReportingYearsHelper.FormatYearAsReportingPeriod(item.Item1)) (@item.Item2 .CSV file)
```

Comment:
Didn't notice this before - previously this didn't have a dot so it was just "CSV file". Very minor but it would be good to not change this unnecessarily, could we get rid of the dot again?